projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_configuration.py

```diff
@@ -180,3 +188,19 @@ def add_definitions(config_builder: ConfigurationBuilder) -> None:
         default_value=None,
         description="The name of the Impala pool to execute queries in.",
     )
+    config_builder.add(
+        key=IMPALA_RETRIES_ON_ERROR,
+        default_value=5,
+        description="The number of times a query will be retried if an exception is raised.",
+    )
+    config_builder.add(
+        key=IMPALA_ERROR_BACKOFF_SECONDS,
+        default_value=30,
+        description=(
+            "Exponential backoff in seconds, in case retries of a failed query are needed. "
+            "Calculated as `2 ** recovery_cursor.get_retries() * backoff_seconds`, where "
```

Comment:
```suggestion
            "Calculated as `2 ** retry-attempt * backoff_seconds`, where "
```

Why mention `recovery_cursor.get_retries()` ? This is implementation detail.